packages/components/Tabs/src/TabsItem.types.ts

```diff
@@ -57,23 +57,6 @@ export interface TabsItemProps extends IPressableProps {
    */
   disabled?: boolean;
 
-  /*
-   ** Defines the current tabs item's position in tabs for accessibility purposes. It's recommended to set this value if
-   ** tabs item are not direct children of tabs. This value is auto-generated if tabs item are direct children of tabs.
-   */
-  accessibilityPosInSet?: number;
-
-  /*
-   ** Defines the number of tabs items in the group for accessibility purposes. It's recommended to set this value if tabs
-   ** items are not direct children of tabs. This value is auto-generated if tabs items are direct children of tabs.
-   */
-  accessibilitySetSize?: number;
-
-  /*
-   ** An accessibility label for narrator.
-   */
-  accessibilityLabel?: string;
-
```

Comment:
@dake3601, @nabil-k, @sriya112 I removed these properties from TabsItem because they're included in IPressableProps